doc/classes/Window.xml

```diff
@@ -469,6 +469,14 @@
 				Sets layout direction and text writing direction. Right-to-left layouts are necessary for certain languages (e.g. Arabic and Hebrew).
 			</description>
 		</method>
+		<method name="set_unparent_when_invisible">
+			<return type="void" />
+			<param index="0" name="unparent" type="bool" />
+			<description>
+				If [param unparent] is [code]true[/code], the window is automatically unparented when going invisible.
+				[b]Note:[/b] Make sure to keep a reference to the node, otherwise it will be orphaned.
```

Comment:
I guess, it should be mentioned that manual `queue_free()` is required as well, not just keeping reference.